Memorize/Screens/ThemeEditor/ThemeEditorViewModel.swift

```diff
@@ -36,4 +36,37 @@ class ThemeEditorViewModel: ObservableObject {
             theme.emojis.remove(at: index)
         }
     }
+
+    func pairsCount() -> Int {
+        var count = 0
```

Comment:
such approach leads to reset explicit number of pars when to 0 when change form explicit to any other and back